include/filters.php

```diff
@@ -210,11 +210,12 @@ public function get_pages( $pages, $args ) {
 			// Take care that 'exclude' argument accepts integer or strings too.
 			$args['exclude'] = array_merge( wp_parse_id_list( $args['exclude'] ), $this->get_related_page_ids( $language, 'NOT IN', $args ) ); // phpcs:ignore WordPressVIPMinimum.Performance.WPQueryParams.PostNotIn_exclude
 			$pages = get_pages( $args );
+			$pages = ! $pages ? array() : $pages;
 		}
 
 		$ids = wp_list_pluck( $pages, 'ID' );
 
-		if ( ! $once ) {
+		if ( ! $once && is_array( $pages ) ) {
```

Comment:
- These 2 modifications look redundant. Am I wrong?
- If you assume that `get_pages()` returns false then something's broken (because we start from a non empty `$pages` and transform it to `false`. So, it would be better to take the opportunity to use an intermediate variable to fix this.

Reply:
- Indeed, it's redundant...
- Good idea to use an intermediate variable, didn't think about overriding `$pages` ...
Fixed in f034b02.